modules/wallet/src/main/scala/org/tessellation/wallet/fileio.scala

```diff
@@ -0,0 +1,32 @@
+package org.tessellation.wallet
+
+import cats.effect.Async
+
+import fs2.io.file.{Files, Flags, Path}
+import fs2.{Stream, text}
+import io.circe.fs2._
+import io.circe.syntax._
+import io.circe.{Decoder, Encoder}
+
+object fileio {
```

Comment:
IMO it should be `package org.tessellation.wallet.file` with `object io`

Reply:
I can do that, however, I don't understand the point of shadowing existing root prefixes.

Reply:
done

Reply:
just to be consistent with namings and packages, no need for reinventing new ones 🤔